100-CNAB.md

```diff
@@ -93,7 +93,6 @@ The process for standardization is described in an appendix:
 ## History
 
 - The `bundle.cnab` is now the name of a signed `bundle.json`.
-- The `bundle.json` is now a stand-alone artifact, not part of the invocation image.
```

Comment:
Rather than rewrite history, I think it would be better to add a new history line. Also, I'm not sure what the deleted line was referring to - perhaps `bundle.json` used to be literally part of the invocation image?

Reply:
@glyn not sure if my added line made it less confusing. Let me know your thoughts or suggestions.

Reply:
The line was fine, but in the wrong place. ;-)